datasync/chaindatafetcher/kas/repository_traces.go

```diff
@@ -0,0 +1,196 @@
+package kas
+
+import (
+	"fmt"
+	"github.com/klaytn/klaytn/blockchain"
+	"github.com/klaytn/klaytn/blockchain/types"
+	"github.com/klaytn/klaytn/blockchain/vm"
+	"github.com/klaytn/klaytn/common"
+	"reflect"
+	"strings"
+)
+
+var emptyTraceResult = &vm.InternalTxTrace{
+	Value: "0x0",
+}
+
+func isInternalTxResult(trace *vm.InternalTxTrace) bool {
+	return !reflect.DeepEqual(trace, emptyTraceResult)
+}
+
+// getMotherTx returns a entry transaction which may call internal transactions.
+func getMotherTx(block *types.Block, txIdx int, tx *types.Transaction) *Tx {
```

Comment:
Is `MotherTx` is a common word? 
How about "ParentTx", "EntryTx" or "Caller"?

Reply:
`OriginalTx`?

Reply:
How about `ExternalTx`?

Reply:
This function gets `*types.Transaction` as an input and returns `*Tx`. 
These two "transactions" mean the same "action" but have different formats. 
I think we better change this function to `convertToKasTx` and name a variable `kasTx`.

Reply:
I think `EntryTx` is the best as its meaning.

Reply:
`kasTx` seems not quite right. Then, the model `Tx` name should be changed. The package name is `kas`, so it is known from outside. :) 

The point is how to distinguish (external?) transaction, internal transaction, and entry (external) transaction. So, I will take `EntryTx`.